### PR TITLE
Sprite sheets for neighborhood and their json mappings

### PR DESCRIPTION
**Pull request by JillianK**
Add sprite sheets with the neighborhood skin assets other than Painter (since those will be treated differently). The category breakdowns are:

Buildings: Blue house, red house, bodega
Vehicles: Food truck, taxi
Sidewalk: just sidewalk
Walls: all the types of walls
Other: benches, landscape, other obstacles

The JSON format is
`{
    "0": {
        "name": "street",
        "sheet": "other.png",
        "row": 0,
        "column": 0
    },
    "1": {
        "name": "N-taxi-2",
        "sheet": "vehicles.png",
        "row": 0,
        "column": 0
    },
...
}`

There was a concern about sprite sheet size for older devices in [this thread](https://codedotorg.slack.com/archives/C01EF4GJ9GE/p1620771779281900?thread_ts=1620758767.272600&cid=C01EF4GJ9GE) but we shouldn't hit the limit 3 megapixels since the largest sheet is 320x576

Here's an example of what the sprite sheets look like: 
buildings

## Links

- sheet used to generate these: [here](https://docs.google.com/spreadsheets/d/1j6r9cWArsQJF8j9sm-87kOXs6oLh7xceHdH86vglBr0/edit#gid=1461173926)
- sprite sheet plan: [here](https://docs.google.com/document/d/1AyJyfEphOqtEaIGl-F6o4efvtmYRfsxVv-em87htMJI/edit#)
- jira ticket: [CSA-289](https://codedotorg.atlassian.net/browse/CSA-289?atlOrigin=eyJpIjoiNzBjYmI5ZTgxZDM5NGY0Zjg3ZTUxODg3MTkzODNmNmMiLCJwIjoiaiJ9)
- Zip file with the scripts I used to generate these and comments on how they work: [in the neighborhood folder in gdrive](https://drive.google.com/file/d/12jtDVxO4G-_cdkHViJijtJ6mxVvB-R8l/view?usp=sharing)

## Follow-up work

- Creating sprite sheets for the painter

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/maze/sprites.json [hunk 1]
```diff
@@ -0,0 +1,1784 @@
...
+        "row": 4,
+        "column": 1
+    },
+    "43": {
+        "name": "S-DonutTruck-8",
+        "sheet": "vehicles.png",
+        "row": 4,
+        "column": 2
+    },
+    "46": {
```
Comment (hannahbergam) [inline apps/src/maze/sprites.json]: Just for my own curiosity, is there any reason 44 and 45 are skipped?

Comment (hannahbergam) [reply apps/src/maze/sprites.json]: Ahh disregard! These id's align with the google sheet, where 44 and 45 are missing. I understand:)

**Review by hannahbergam — COMMENTED** [review]
This looks sweeeet great job! I presume you already spoke with the crew about where this json file should live. That said, because it only contains neighborhood sprites, could it be called something more specific like neighborhoodSprites.json?

**Review by molly-moen — APPROVED** [review]
this is awesome, good job!

**Review by jmkulwik — COMMENTED** [review]
Looks awesome! Just a reminder: add the scripts to the https://github.com/code-dot-org/code-dot-org/tree/staging/tools/scripts folder.

**Pull request merged**